src/components/AppInvalidGuest.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <Content app-name="cookbook">
+        <!-- <AppNavi class="app-navigation" /> -->
+        <AppContent>
+            <div class="main">
+                <div class="dialog">
+                    <div class="message">
+                        {{ t('cookbook', 'Cannot access recipe folder.') }}
+                    </div>
+                    <div class="explanation">
+                        {{
+                            t('cookbook', 'You are logged in with a guest account. Therefore, you are not allowed to generate arbitrary files and folders on this nextcloud instance. To be able to use the cookbook app as a guest account, you need to specify a folder where all recipes are stored. You will need write permission to this folder.')
+                        }}
+                    </div>
+                    <div class="explanation">
```

Comment:
class `explanation` is not used and may be removed
```suggestion
                    <div>
```